packages/wrangler/src/__tests__/api/startDevWorker/BundleController.test.ts

```diff
@@ -68,20 +67,15 @@ describe("happy path bundle + watch", () => {
 		});
 		const config: StartDevWorkerOptions = {
 			name: "worker",
-			script: unusable(),
-			_entry: {
-				file: path.resolve("src/index.ts"),
-				directory: path.resolve("src"),
-				format: "modules",
-				moduleRoot: path.resolve("src"),
-				name: "worker-name",
-			},
-			_additionalModules: [],
+			script: { path: path.resolve("src/index.ts") },
```

Comment:
As a user I would expect this filepath to be relative to the cwd. So the user shouldn't have to call path.resolve – the ConfigController should do that on their behalf